+neurostim/+stimuli/movie.m

```diff
@@ -73,7 +73,15 @@ function beforeTrial(o)
                 file = fullfile(o.path,o.filename);
             end
             if strcmpi(o.currentlyLoadedFile,file)
-                % No need to reload
+                % No need to reload images
```

Comment:
Thi reloading should not be necessary. 
What is happening is that the properties are reset to their default value (i.e. the first value ever assigned) at the start of trial 2. That is why they are empty.

The solution is not to reload the file, but to make the parameters 'Sticky'. 
Please have a look at the neurostim.parameter object to see how this is defined, then adjust the addProperty calls 
in the constructor with the 'sticky' flag.  After that this code can be removed.